src/pages/guides/data-fetching.md

```diff
@@ -3,27 +3,62 @@ layout: ~/layouts/Main.astro
 title: Data Fetching
 ---
 
-Astro components and pages can fetch remote data to help generate your pages. Astro provides two different tools to pages to help you do this: **fetch()** and **top-level await.** 
+Astro components and pages can fetch remote data to help generate your pages. Astro provides two different tools to pages to help you do this: **fetch()** and **top-level await.**
 
-### `fetch()`
+## `fetch()`
 
-Astro pages have access to the global `fetch()` function in their setup script. `fetch()` is a native JavaScript API ([MDN](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)) that lets you make HTTP requests for things like APIs and resources. 
+Astro pages have access to the global `fetch()` function in their setup script. `fetch()` is a native JavaScript API ([MDN](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)) that lets you make HTTP requests for things like APIs and resources.
 
 Even though Astro component scripts run inside of Node.js (and not in the browser) Astro provides this native API so that you can fetch data at page build time.
 
-
 ```astro
+// Movies.astro
 ---
-const response = await fetch('http://example.com/movies.json');
+const response = await fetch('https://example.com/movies.json');
```

Comment:
can we use a real API here, so that this works if a user copy-pasted? I've used https://pokeapi.co/ in the past:

```
const response = await fetch('https://pokeapi.co/api/v2/pokemon?limit=150');
```

(I know you didn't write this originally, but would love this change if possible!)

Reply:
I'll merge now and leave this for a later PR, since I'd rather take the time to make the example fully copy-pastable